cmd/manager/main.go

```diff
@@ -144,6 +144,8 @@ func main() {
 		log.Info("Could not create metrics Service", "error", err.Error())
 	}
 
+	/*
```

Comment:
You should disable serveCRMetrics and CreateMetricsService either

Reply:
you right
Done, please check again